pkg/kvstore/backend.go

```diff
@@ -21,6 +21,11 @@ import (
 	"google.golang.org/grpc"
 )
 
+const (
+	// RateLimitOption specifies maximum kv operations per second
+	RateLimitOption = "kv.qps"
```

Comment:
Please consider having 2 options called `etcd.qps` and `consul.qps`. It will be coherent with all other options that we have for both kvstores

Reply:
This is the only option that would be shared between these two, but I think there is value in replicating them so they follow this convention.